### PR TITLE
First draft for new rule on usage of absolute URI (#249)

### PR DESCRIPTION
**Pull request by meshcalero**

### CODE REVIEW

**Comment by tfrauenstein** [comment]
👍

**Comment by codekultur** [comment]
👍

File: chapters/hyper-media.adoc [hunk 1]
```diff
@@ -63,6 +63,18 @@ experience and present your findings in the
 https://techwiki.zalando.net/display/GUL/API+Guild[API Guild [internal
 link]].
 
+[#215]
+== {MUST} Use full, absolute URI
```
Comment (whiskeysierra) [inline chapters/hyper-media.adoc]: Is this the official wording from the RFCs about those kinds of URIs?

**Comment by tkrop** [comment]
👍

**Comment by ePaul** [comment]
:+1:

**Pull request merged**